com.ca.lsp.cobol/lsp-core-cobol-parser/src/test/java/com/ca/lsp/core/cobol/preprocessor/sub/line/rewriter/impl/CobolLineIndicatorProcessorImplTest.java

```diff
@@ -0,0 +1,169 @@
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Broadcom, Inc. - initial API and implementation
+ */
+package com.ca.lsp.core.cobol.preprocessor.sub.line.rewriter.impl;
+
+import static com.ca.lsp.core.cobol.preprocessor.CobolSourceFormat.FIXED;
+import static com.ca.lsp.core.cobol.preprocessor.ProcessingConstants.COMMENT_TAG;
+import static com.ca.lsp.core.cobol.preprocessor.sub.CobolLineTypeEnum.COMMENT;
+import static com.ca.lsp.core.cobol.preprocessor.sub.CobolLineTypeEnum.COMPILER_DIRECTIVE;
+import static com.ca.lsp.core.cobol.preprocessor.sub.CobolLineTypeEnum.CONTINUATION;
+import static com.ca.lsp.core.cobol.preprocessor.sub.CobolLineTypeEnum.NORMAL;
+import static com.ca.lsp.core.cobol.preprocessor.sub.CobolLineTypeEnum.DEBUG;
+import static com.ca.lsp.core.cobol.preprocessor.ProcessingConstants.WS;
+import static org.junit.Assert.assertEquals;
+
+import com.ca.lsp.core.cobol.preprocessor.sub.CobolLine;
+
+import java.util.Arrays;
+import java.util.List;
+import org.junit.Test;
+
+public class CobolLineIndicatorProcessorImplTest {
+
+  /**
+   * Testing preprocessing formatting for COBOL files, making sure that only needed information is
+   * being passed to the parser to avoid issues and making token recognition easier
+   */
+  @Test
+  public void processLinesTest() {
+    final String EMPTY_STRING = "";
+    CobolLine continuationLine = new CobolLine();
+    CobolLine trailingQuoteContinuationLine = new CobolLine();
+    CobolLine emptyContinuationLineToTest = new CobolLine();
+    CobolLine singleContinuationLineToTest = new CobolLine();
+    CobolLine goodContinuationLineToTest = new CobolLine();
+    CobolLine badContinuationLineToTest = new CobolLine();
+    CobolLine commentLineToTest = new CobolLine();
+    CobolLine continuationLineWithEndQuotes = new CobolLine();
+    CobolLine compilerDirectiveLineToTest = new CobolLine();
+    CobolLine normalLineToTest = new CobolLine();
+    CobolLine debugLineToTest = new CobolLine();
+    continuationLine.setType(CONTINUATION);
+    continuationLine.setIndicatorArea("-");
+    continuationLine.setContentAreaA("        \"CONTINUED LINE ENDS HERE\"     ");
+
+    goodContinuationLineToTest.setType(CONTINUATION);
+    goodContinuationLineToTest.setSuccessor(continuationLine);
+    goodContinuationLineToTest.setIndicatorArea(WS);
+    goodContinuationLineToTest.setContentAreaA("       \"RANDOM TEXT   ");
+    goodContinuationLineToTest.setContentAreaB("        ");
+
+    singleContinuationLineToTest.setType(CONTINUATION);
+    singleContinuationLineToTest.setSuccessor(continuationLine);
+    singleContinuationLineToTest.setIndicatorArea(WS);
+    singleContinuationLineToTest.setContentAreaA("       'RANDOM TEXT   ");
+    singleContinuationLineToTest.setContentAreaB("        ");
```

Comment:
Do we really need to emphasize this "to test"?

Reply:
Removed